### ISSUE TITLE
Identity crashes on server load.

### ISSUE THREAD

**Issue by Charxter** [Draylar/identity#200]
I have all of my mods properly installed, I have the fabric mod server properly setup. Single player identity works perfectly fine but anytime I join my server I get to "Joining World" and then it says "Saving World," closing my game shortly after.

LOG: https://pastebin.com/0hD35x1n
(Minecraft 1.16.5, Fabric API 0.34.0, Fabric Launcher 1.13.0)

**Comment by James103** [comment]
Can you please downgrade Fabric API to 0.33.x then restart the server?

**Comment by Charxter** [comment]
> Can you please downgrade Fabric API to 0.33.x then restart the server?

That worked up until it said incomplete set of tags but I have everything properly setup so I don't understand why thats happening

**Comment by James103** [comment]
> > Can you please downgrade Fabric API to 0.33.x then restart the server?
> 
> That worked up until it said incomplete set of tags but I have everything properly setup so I don't understand why thats happening

Can you please post the resulting game logs (using Pastebin, hastebin, paste.ee, etc)?

**Comment by Charxter** [comment]
> > > Can you please downgrade Fabric API to 0.33.x then restart the server?
> > 
> > 
> > That worked up until it said incomplete set of tags but I have everything properly setup so I don't understand why thats happening
> 
> Can you please post the resulting game logs (using Pastebin, hastebin, paste.ee, etc)?

Here's my Minecraft log: https://pastebin.com/2FpSwVPk

**Comment by James103** [comment]
The server likely does not have the Identity mod installed, hence the missing server tags. Are you able to connect to the server normally without having the Identity mod installed?

**Comment by Fauli1221** [comment]
Identity or fabric api 34 is most likely the issue

**Comment by Charxter** [comment]
> The server likely does not have the Identity mod installed, hence the missing server tags. Are you able to connect to the server normally without having the Identity mod installed?

I *do* have identity in my server files, same with my client files.
![image](https://user-images.githubusercontent.com/83691950/117109595-e7a1db80-ad52-11eb-9e12-08d052d8153a.png)
![image](https://user-images.githubusercontent.com/83691950/117109623-ee305300-ad52-11eb-929a-7bb5483cd777.png)
(Ignore the files I have set to .jar.x, those are disabled since I am not using them)

I did just load up Minecraft with Identity disabled and my F3 screen said I was on a "vanilla" server even though I know I'm not?
Just reset something in my server files & rebooting Minecraft. Waiting for that before I post this comment.

I figured out my issue, I had my launch jar set to "server.jar" instead of "fabric-server-launch.jar" which made the game think I was going for vanilla Minecraft.

TL;DR Figured out my issue. Thanks for helping out!
